tests/wps_restapi/test_processes.py

```diff
@@ -160,6 +160,7 @@ def test_get_processes(self):
             assert "version" in process and isinstance(process["version"], str)
             assert "keywords" in process and isinstance(process["keywords"], list)
             assert "metadata" in process and isinstance(process["metadata"], list)
+            assert len(process["jobControlOptions"]) == 1 and 'async-execute' in process["jobControlOptions"]
```

Comment:
please use the variable instead of the string

Reply:
Solved, see last commit